examples/metrics/collector.py

```diff
@@ -21,11 +21,10 @@
 from opentelemetry.ext.otcollector.metrics_exporter import (
     CollectorMetricsExporter,
 )
-from opentelemetry.sdk.metrics import Counter, MeterProvider
+from opentelemetry.sdk.metrics import Counter
 from opentelemetry.sdk.metrics.export.controller import PushController
 
 # Meter is responsible for creating and recording metrics
-metrics.set_preferred_meter_provider_implementation(lambda _: MeterProvider())
 meter = metrics.get_meter(__name__)
```

Comment:
This returns an api `DefaultMeter` now instead of a sdk `Meter`, shouldn't we have this line above?

```python
metrics.set_meter_provider(sdk.metrics.MeterProvider())
```

Or is it intentional that the examples only use the API types? I think it'd be more helpful to show using the SDK since that's how most users will do it.